.github/workflows/mlkem.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     defaults:
       run:
         shell: bash
-        working-directory: libcrux-ml-kem
+        working-directory: libcrux-ml-dsa
```

Comment:
This looks wrong. You probably wanted to do this in `mldsa.yml` 🙂

Reply:
Ugh, sorry about that. Fixing